packages/core/TextSearch/TextSearchManager.ts

```diff
@@ -45,7 +45,7 @@ export default class TextSearchManager {
         const { AdapterClass } = this.pluginManager.getTextSearchAdapterType(
           adapterConfig.type,
         )
-        const adapter = new AdapterClass(adapterConfig, this.pluginManager)
+        const adapter = new AdapterClass(adapterConfig)
```

Comment:
probably would want this to say new AdapterClass(adapterConfig, undefined, pluginManager)